tests/specs/future/runtime_api/main.js

```diff
@@ -9,6 +9,7 @@ console.log("Deno.ftruncate is", Deno.ftruncate);
 console.log("Deno.ftruncateSync is", Deno.ftruncateSync);
 console.log("Deno.flock is", Deno.flock);
 console.log("Deno.flockSync is", Deno.flockSync);
+console.log("Deno.FsFile.prototype.rid is", Deno.FsFile.prototype.rid);
```

Comment:
Since this relies on actual instance of `Deno.FsFile`, can you open any file here and assert that the instance doesn't have this field?

Reply:
Done.